### ISSUE TITLE
hook not successfully when symbols is lazy bindings

### ISSUE THREAD

**Issue by boalove** [facebook/fishhook#10]
I use it in mac os x, as dyld_interposing does not works sometimes.
I used it the hook a system api(LSOpenCFURLRef), but this symbol is lazy binding.
And I cannot hook it successfully.
Any idea?
Thanks!

**Comment by grp** [comment]
Yeah, this is an issue with fishhook. For that, I'd suggest checking out @landonf's PLPatchMaster — it does a lot more, but it includes hooking lazy/indirect symbols.
